plugins/lxd.plugin

```diff
@@ -4,6 +4,10 @@
 init_deployment_lxd()
 {
   init_deployment_dir
+  # Minimize per default, if no specified otherwise
```

Comment:
This will not deploy man pages, package documentation and locales per default and greatly reduce the image size.

See: https://github.com/molior-dbs/molior-tools/blob/master/molior-deploy#L752